nixos/modules/testing/service-runner.nix

```diff
@@ -107,10 +109,9 @@ let
   opts = { config, name, ... }: {
     options.runner = mkOption {
     internal = true;
-    description = lib.mdDoc ''
-        A script that runs the service outside of systemd,
-        useful for testing or for using NixOS services outside
-        of NixOS.
+    description = mdDoc ''
+      A script that runs the service outside of systemd, useful for testing or
+      for using NixOS services outside of NixOS.
     '';
```

Comment:
We should not change formatting that changes the code to avoid unnecessary possibilities  regressions.